webpack.config.js

```diff
@@ -28,8 +28,10 @@ const hubspotConfig = ({ portal, autoupload } = {}) => {
         {
           test: /\.s[ac]ss$/i,
           use: [
+            "style-loader",
```

Comment:
I don't think that this is necessary and I think that the `MiniCssExtractPlugin.loader` needs to be first in the list of loaders so that it is executed last.